docs/modules/operation/pages/quick-start/notification-config.adoc

```diff
@@ -11,13 +11,18 @@ We assume that you have followed the previous steps and are ready to proceed to
 Now that you have xref:operation:quick-start/baseline.adoc[established your environment's baseline], it's time to take your notes and configure your custom notifications.
 Establishing notifications that are tailored for your environment and monitoring requirements lets the right people see and respond to the right issues in a timely manner.
 
-Here, we will describe how to configure event notifications and path outages.
-Event notifications can be configured to provide UEI-specific coverage, allowing for granular control of incoming information.
-Path outages prevent the generation of excess notifications by identifying network paths where failures may cause multiple nodes to appear offline.
+Here, we will describe how to configure event notifications.
+You can configure event notifications to provide coverage of specific unique event identifiers (UEIs), allowing for granular control of incoming information.
+
+You can define multiple notifications for the same UEI to use different destination paths.
+By default, {page-component-title} uses the destination path of all notifications matching the event's UEI.
+You can configure notifd to process only the destination path of the first matching notification by editing `$\{OPENNMS_HOME}/etc/notifd-configuration.xml` and setting `match-all="false"` in the top-level `notifd-configuration` element.
+
+NOTE: Changes to `$\{OPENNMS_HOME}/etc/destinationPaths.xml` happen immediately, with no need to restart {page-component-title}.
 
 == Configure event notifications
 
-An event notification defines a destination path and message template for a specific event UEI.
+An event notification defines a destination path and message template for a specific UEI.
 It is possible to define multiple notifications for the same UEI to use different filters, destination paths, and message templates.
 Follow these steps to configure event notifications:
```

Comment:
There's another instance of the gear symbol here, at line 29, that I can't get to.